### ISSUE TITLE
Addon-A11y: add blurred vision filter

### ISSUE THREAD

**Issue by yannbf** [storybookjs/storybook#12730]
**Is your feature request related to a problem? Please describe**
Google chrome has recently released the ability to [emulate vision deficiencies](https://youtu.be/MLNERCykDPg?t=34), and there's one type which addon-a11y does not support yet which is blurred vision. You can see it in effect in this video: https://streamable.com/tzwyvh.

Here's an example of Chrome's rendering emulation:
![image](https://user-images.githubusercontent.com/1671563/95662131-0df0f700-0b35-11eb-8893-401dbe2e5b18.png)

I think this feature brings value and it's a nice addition to the addon! In the video you can see how the blurred effect helps identify if the image is still somewhat understandable for users with blurred vision. It also allows to understand wether the design presents a success or failure intent even blurred, and other cases that can be read [here](https://uxdesign.cc/the-blurry-eye-test-1bd12bc6c3f8).

**Are you able to assist to bring the feature to reality?**
yes, PR on the way!

**Comment by shilman** [comment]
Egads!! I just released https://github.com/storybookjs/storybook/releases/tag/v6.1.0-alpha.23 containing PR #12731 that references this issue. Upgrade today to try it out!

You can find this prerelease on the `@next` NPM tag.

Closing this issue. Please re-open if you think there's still more to do.
